AutomatedTesting/Gem/PythonTests/Prefab/tests/PrefabTestUtils.py

```diff
@@ -251,16 +251,17 @@ def validate_spawned_entity_transform(entity, expected_position, expected_rotati
     spawned_entity_transform = entity.get_world_translation()
     spawned_entity_rotation = entity.get_world_rotation()
 
-    x_rotation_success = math.isclose(spawned_entity_rotation.x, expected_rotation.x,
-                                      rel_tol=1e-5)
-    y_rotation_success = math.isclose(spawned_entity_rotation.y, expected_rotation.y,
-                                      rel_tol=1e-5)
-    z_rotation_success = math.isclose(spawned_entity_rotation.z, expected_rotation.z,
-                                      rel_tol=1e-5)
+    position_success = wait_for_condition(lambda: spawned_entity_transform == expected_position, 3.0)
+    x_rotation_success = wait_for_condition(lambda: math.isclose(spawned_entity_rotation.x, expected_rotation.x,
+                                                                 rel_tol=1e-5), 3.0)
+    y_rotation_success = wait_for_condition(lambda: math.isclose(spawned_entity_rotation.y, expected_rotation.y,
+                                                                 rel_tol=1e-5), 3.0)
+    z_rotation_success = wait_for_condition(lambda: math.isclose(spawned_entity_rotation.z, expected_rotation.z,
+                                                                 rel_tol=1e-5), 3.0)
```

Comment:
would an local function that AND's the rotation x,y,z checks allow for faster fail time. currently this could take up to 9 seconds to fail. you could reduce that to 3 seconds by having rotation_success = wait_for_condition(lambda: aggregate_rotation_function(spawned_entity_rotation, expected_rotation), 3.0)

Reply:
You're right, it could. I'll look into splitting it out into an aggregate.

Reply:
Done.